src/apps/bristleback_apps/bm_rbr/CMakeLists.txt

```diff
@@ -280,6 +282,14 @@ set(APP_DEFINES
     APP_NAME="${APP_NAME}"
     CFG_TUSB_MCU=OPT_MCU_STM32U5
 
+    # Uncomment/change these to enable tracing!
+    # TRACE_ENABLE
+    # TRACE_BUFF_LEN=1024
+    # TRACE_TASKS
+    # TRACE_USE_COREDEBUG
+    # TRACE_ISR
+    # TRACE_SERIAL
+
```

Comment:
These are used to enable tracing and must be uncommented in addition to the trace.c file being included.

Reply:
Can you add a comment pointing at `# ${SRC_DIR}/lib/common/trace.c`?

Reply:
Done